packages/shopify-codemod/transforms/global-reference-to-import.js

```diff
@@ -68,6 +69,13 @@ export default function globalReferenceToImport(
     return fileForIdentifier[identifier];
   }
 
+  function searchForNamedExports(filename) {
+    if (filename.endsWith('.coffee'))
+      return false;
+    const namedExports = j(fs.readFileSync(filename).toString())
```

Comment:
If the transform happens to miss a case due to not using the most accurate way of validating the presence of a named export, wouldn't that result in a wrong import and therefore broken code coming out of the transform?

Why is the performance more important than the integrity of the transform? We only run these transforms once on each file anyways right? Wouldn't we want that 1 time that it's run to be longer but have confidence in the output rather than having the 1 time it's run to be quick but having to double check the result manually to ensure no imports are wrong?